src/System.IO.FileSystem.Watcher/src/System/IO/FileSystemWatcher.cs

```diff
@@ -697,4 +711,57 @@ private bool IsSuspended()
             return _initializing || DesignMode;
         }
     }
+
+    internal sealed class NormalizedFilterCollection : Collection<string>
+    {
+        public NormalizedFilterCollection()
+        {
+        }
+
+        public NormalizedFilterCollection(IList<string> list) : base(list)
```

Comment:
As stated, you don't need this yet. We shouldn't have the constructor overload as discussed in the API review.